README.md

```diff
@@ -117,47 +120,171 @@ If you'd like to get a message when the button is pressed or released, call the
     {:gpio_interrupt, 17, :falling}
     :ok
 
-Note that after calling `set_int`, the calling process will receive an initial message with the state of the pin.
-This prevents the race condition between getting the initial state of the pin and turning on interrupts. Without it, you could get the state of the pin, it could change states, and then you could start waiting on it for interrupts. If that happened, you would be out of sync.
+Note that after calling `set_int`, the calling process will receive an initial 
+message with the state of the pin.  This prevents the race condition between 
+getting the initial state of the pin and turning on interrupts. Without it, 
+you could get the state of the pin, it could change states, and then you could 
+start waiting on it for interrupts. If that happened, you would be out of sync.
 
 ## SPI
 
-A SPI bus is a common multi-wire bus used to connect components on a circuit
-board. A clock line drives the timing of sending bits between components. Bits
-on the `MOSI` line go from the master (usually the processor running Linux) to
-the slave, and bits on the `MISO` line go the other direction. Bits transfer
-both directions simultaneously. However, much of the time, the protocol used
-across the SPI bus has a request followed by a response and in these cases, bits
-going the "wrong" direction are ignored.
-
-The following shows an example ADC that reads from either a temperature sensor
-on CH0 or a potentiometer on CH1.
+A Serial Peripheral Interface (SPI, pronounced "spy") bus is a common multi-wire 
```

Comment:
I'm not sure if there's an official pronunciation. I know people who say "spy" and "s-p-i". I2C is similar: some say "eye-two-see" and others say "eye-squared-see". I switch back and forth depending on who I'm talking to and I've even had people correct me. My suggestion is to leave the pronunciation part out, but to link "Serial Peripheral Interface" to the Wikipedia article (https://en.wikipedia.org/wiki/Serial_Peripheral_Interface_Bus).